bbc/simorgh
# WSTEAM1-280: Add media-page identifier to ATI tracking.

Resolves JIRA [WSTEAM1-280](https://jira.dev.bbc.co.uk/browse/WSTEAM1-280)

![Screenshot 2023-03-07 at 13 03 05](https://user-images.githubusercontent.com/32307964/223435881-124e0405-79fc-4b93-a4de-399d20c099f0.png)

![Screenshot 2023-03-07 at 13 03 17](https://user-images.githubusercontent.com/32307964/223435884-7c90c29c-27a9-4327-8ab3-3f99cb9ab554.png)

![Screenshot 2023-03-07 at 13 03 21](https://user-images.githubusercontent.com/32307964/223435886-cfa9f39e-4a4e-4b79-87de-5b75d68b7812.png)

Overall changes
======
Adds media-page identifier to ATI so that MediaPage sends tracking information adjacent to that of Article Pages. 

Code changes
======

- _ATI Tracking functions for building the Article URL has been modified to accommodate the media-article page type._
- _EventTracking has been modified to accommodate tracking requests for the media-article page type._

Testing
======
1. Open a MediaArticlePage.
2. Open the network tab in chrome. 
3. ATI tracking requests should be made similar to that of ArticlePages, but with 'media-article' set as the x7 header. 

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
